### PR TITLE
layout: Twitter summary card

### PR DESCRIPTION
**Pull request by pchaigno**
This pull request attempts to add a summary card for Twitter, to get a basic preview with the title and logo.

### CODE REVIEW

File: sphinx_rtd_theme/layout.html [hunk 1]
```diff
@@ -112,6 +112,11 @@
   {%- endif %}
   {%- endblock %}
   {%- block extrahead %} {% endblock %}
+
+  <meta name="twitter:card" content="summary"/>
+  <meta name="twitter:site" content="@ciliumproject"/>
+  <meta name="twitter:title" content="{{ title|striptags|e }}{{ titlesuffix }}"/>
+  <meta name="twitter:image" content="{{ canonical_url }}{{ pathto('_static/' + logo, 1) }}"/>
```
Comment (pchaigno) [inline sphinx_rtd_theme/layout.html:119]: @genbit Is this the proper way to get the full URL to the docs? I see that e.g. on line 175 we hardcode the URL, so maybe I should just do that.

Comment (genbit) [reply sphinx_rtd_theme/layout.html:119]: @pchaigno I'm not sure actually. We can try deploying it on readthedocs and see if it works.

Comment (pchaigno) [reply sphinx_rtd_theme/layout.html:119]: Test in production? One new thing off my bucket list :ballot_box_with_check:

Comment (pchaigno) [reply sphinx_rtd_theme/layout.html:119]: @genbit Should we merge this then?

Comment (genbit) [reply sphinx_rtd_theme/layout.html:119]: done

**Pull request merged**